docs/index.rst

```diff
@@ -10,6 +10,66 @@ parallelize, Just-In-Time compile to GPU/TPU, and more.
    Please see `Distributed arrays and automatic parallelization <https://jax.readthedocs.io/en/latest/notebooks/Distributed_arrays_and_automatic_parallelization.html>`_ tutorial and the :ref:`jax-array-migration`
    guide for more information.
 
+Key Ideas of Jax
+================
+.. grid:: 3
+
+   .. card:: 
+
+      Familiar API
```

Comment:
I think you can do something like
```
.. card:: Familiar API

   JAX provides a familiar NumPy-style API for ease of adoption by researchers and engineers. 
```
to make the title recognized as a title

Reply:
Tried that. thought the horizontal line looked nice as a separator, though the bolding is nice too. Here they are side by side.

I don't have a strong preference. Do you like one of the two better? Happy to change it

<img width="826" alt="image" src="https://user-images.githubusercontent.com/7213793/211118231-1801c615-c6ad-496f-94e1-4aa6ffdc6cbe.png">

Reply:
We're into aesthetic bikeshedding now, but my preference would be to make them all like the one on the right, and with the borders removed if possible

Reply:
Nothing a bit of css cant fix! Hows this look? And thank you for the rapid feedback, well converge much faster!

<img width="833" alt="image" src="https://user-images.githubusercontent.com/7213793/211119420-3a5cc921-3cde-453a-8db3-545bf27a92c5.png">

Reply:
Looks great to my eyes 😁

Reply:
🎉 Pushed a latest set of commits to fix the build error. Want to be sure we get this done by monday for that meeting. 

Thanks again for rapid reviews